.eslintrc

```diff
@@ -19,7 +19,7 @@
     "plugin:react/recommended",
     "plugin:storybook/recommended",
     "plugin:testing-library/react",
-    "plugin:@chanzuckerberg/stories/recommended"
+    "plugin:@chanzuckerberg/stories/strict"
```

Comment:
Let me know if you see any oddities with the no-csf-v2 and csf-v3-types rules.

Reply:
Sounds good, will do